src/App.tsx

```diff
@@ -3,29 +3,57 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api/api';
 
 interface State {
   selectedUserId: number;
+  todos: Todo[];
 }
 
 class App extends React.Component<{}, State> {
   state: State = {
     selectedUserId: 0,
+    todos: [],
+  };
+
+  async componentDidMount() {
+    const todos = await getTodos();
+
+    this.setState({ todos });
+  }
+
+  selectUser = (userId: number) => {
+    this.setState({
+      selectedUserId: userId,
+    });
+  };
```

Comment:
```suggestion
    this.setState({ selectedUserId: userId });
  };
```
Думаю, что лучше так